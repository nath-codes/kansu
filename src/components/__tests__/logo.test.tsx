import React from 'react'

import Logo from '../logo'
import { renderWithTheme } from './helpers'

describe('Logo', () => {
  it('renders correctly', () => {
    const src = 'Footer.jpg'
    const tree = renderWithTheme(<Logo src={src} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
