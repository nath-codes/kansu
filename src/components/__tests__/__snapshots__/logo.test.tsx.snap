// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
.c0 {
  height: 3.375rem;
}

<div
  className="c0"
>
  <img
    alt="Kansu logo"
    src="Footer.jpg"
  />
</div>
`;
